### DISCUSSION TITLE
Can I prevent updates for a package that is pinned?

### DISCUSSION THREAD

**Discussion by maxrzaw** [mason-org/mason.nvim#1506] [Q&A]
I have a package that I would like to keep at a specific version. In my config I have:

``` lua
ensure_installed = {
    "angular-language-server@15.2.1",
}
```

This works for keeping the package at the correct version when it is initially installed. My problem is that when I use `U` to update all packages, it upgrades the angular-language-server to the latest version. Is there a way to prevent this from happening for packages that are pinned in ensure installed?

**Comment by aborzunov** [comment]
Did you find a solution?

**Reply by maxrzaw** [reply]
Nope
